front/pages/w/[wId]/ds/new.tsx

```diff
@@ -353,8 +354,8 @@ export default function DataSourceNew({
                 <div className="flex">
                   <Button
                     disabled={disabled || creating}
-                    onClick={() => {
-                      handleCreate();
+                    onClick={async () => {
```

Comment:
That seems fine if it makes TS happy?

Reply:
If the goal is solely to make TS happy, we could simply do `void handleCreate()` instead of making the function an async function + awaiting. But definitely nitpicking here